presto-native-execution/presto_cpp/main/PrestoExchangeSource.h

```diff
@@ -65,6 +74,11 @@ class PrestoExchangeSource : public velox::exec::ExchangeSource {
   // Returns a shared ptr owning the current object.
   std::shared_ptr<PrestoExchangeSource> getSelfPtr();
 
+  // Tracks the currently node-wide queued memory usage in bytes.
+  static inline std::atomic<int64_t> currQueuedMemoryBytes_{0};
+  // Records the node-wide peak queued memory usage in bytes.
+  static inline std::atomic<int64_t> peakQueuedMemoryBytes_{0};
+
```

Comment:
We probably shall do similar things of what fb recommend if we want to go static
```
static std::atomic<int64_t>& currQueuedMemoryBytes() {
  static std::atomic<int64_t> currQueuedMemoryBytes_{0};
  return currQueuedMemoryBytes_;
}